topics/proteomics/tutorials/multiGSEA-tutorial/tutorial.md

```diff
@@ -0,0 +1,117 @@
+
+## Get data
+
+> ### Data Upload
+>
+><hands-on-title> Getting datasets </hands-on-title>
+> 1. Create a new history for this tutorial.
+    {% snippet faqs/galaxy/histories_create_new.md %}
+> 2. Import the datasets from [Zenodo]({{ page.zenodo_link }})  into your Galaxy instance:
+>   - **transcriptomics.tsv**
+>   - **proteomics.tsv**
+>   - **metabolomics.tsv**
+>
+> <comment-title>URLs of the files</comment-title>
+>   - **transcriptomics.tsv** https://zenodo.org/api/records/14216972/files/transcriptome.tsv/content
+>   - **proteomics.tsv**    https://zenodo.org/api/records/14216972/files/proteome.tsv/content
+>   - **metabolomics.tsv**  https://zenodo.org/api/records/14216972/files/metabolome.tsv/content
+>  
+{: .comment}
+{: .hands_on}
+
+# Running MultiGSEA
+
+In this step, you'll use the MultiGSEA tool to perform GSEA-based pathway enrichment on the uploaded datasets.
+
+>## Selecting parameters
+><hands-on-title> Task description </hands-on-title>
+>
+> 1. Select the tool {% tool [multiGSEA](toolshed.g2.bx.psu.edu/repos/iuc/multigsea/multigsea/1.12.0+galaxy0) %} in Galaxy.
+> 2. Configure the input parameters as follows:
+>    - *"Select transcriptomics data"*: `Enabled`
+>        - {% icon param-file %} *"Transcriptomics data"*: `Transcriptomics`
+>    - *"Select proteomics data"*: `Enabled`
+>        - {% icon param-file %} *"Proteomics data"*: `Proteomics`
+>    - *"Select metabolomics data"*: `Enabled`
+>        - {% icon param-file %} *"Metabolomics data"*: `Metabolomics`
+> 3. You can also choose the Gene ID format for every data set. In this tutorial we will use the preset "SYMBOL" for transcriptomics and proteomics. For metabolomics we use HMDB.
+> 4. Select in **Supported organisms** the organism of which the data is about. In our case we select `Homo sapiens (Human)`.
+> 5. **Pathway databases**: Databases often contain their own format in which pathway definitions are provided. So you can select a relevant database. For the tutorial we choose `KEGG`
+> 6. **Combine p-values method**: Choose a method (here `Stouffer` for balanced weighting). To more comprehensively measure a pathway response, multiGSEA provides different approaches to compute an aggregated p value over multiple omics layers. Because no single approach for aggregating p values performs best under all circumstances, Loughin proposed basic recommendations on which method to use depending on structure and expectation of the problem. If small p values should be emphasized, Fisher’s method should be chosen. In cases where p values should be treated equally, Stouffer’s method is preferable. If large p values should be emphasized, the user should select Edgington’s method. Figure 2 indicates the difference between those three methods.
```

Comment:
I reformatted the hands-on boxes a bit. The hands-on boxes should be very concise, just telling the user how to configure the tool. I have moved all your (very useful!) explanations to a tip box inside the hands-on box, but you could also just put them in normal text before or after the hands-on box, as you prefer